WoWPro/WoWPro_Config.lua

```diff
@@ -774,6 +774,21 @@ local function createMainConfig()
                     end,
                 width = "full"
             },
+            doDungeons = {
+                order = 24,
+                type = "toggle",
+                name = L["Enable Dungeon Quests"],
+                desc = L["Skips dungeon-specific quests outside Dungeon guides."],
+                get = function(info) return WoWProCharDB.EnableDungeons end,
+                set = function(info,val)
+                        if WoWProCharDB.EnableDungeons then
+                            WoWProCharDB.EnableDungeons = false
+                        else
+                            WoWProCharDB.EnableDungeons = true
+                        end
+                    end,
+                width = "full"
```

Comment:
remove this line of code and from all the preceeding options within this category to align